src/Raven.Server/Documents/Indexes/Index.cs

```diff
@@ -524,6 +534,17 @@ protected void ExecuteIndexing()
                             Log.ErrorException($"Could not update stats for '{Name} ({IndexId})'.", e);
                         }
 
+                        _indexingInProgress = false;
+
+                        try
+                        {
+                            MaybePersistQueryingTime();
```

Comment:
Statistics for static indexes, but I also thought that maybe we can introduce some mechanism that will create 'action-notifications' e.g. 'We have detected that your (static) index 'People/ByName' haven't been queried for 72 hours Maybe you should remove it?. (Two buttons now) Remove and Dismiss'. Thoughts?